eng/targets/Helix.targets

```diff
@@ -10,14 +10,12 @@
 
   <PropertyGroup Condition="'$(TestDependsOnPlaywright)' == 'true'">
     <SkipHelixQueues>
-      $(HelixQueueAlpine312);
+      $(HelixQueueAlpine314);
```

Comment:
Worth removing this line and running `aspnetcore-helix-matrix` again❔ It's unclear how to determine whether Playwright is working everywhere given all or most Playwright tests are quarantined and at least the `BlazorWasmTemplateTest` tests mostly fail in any case. @javiercn ❔

Reply:
@wtgodbe you mentioned `Alpine 3.14 image doesn't work with Playwright` in #36032. How did you confirm that❔

Reply:
We can basically safely ignore playwright now, I believe the only place that's running now is in the components pipleine, @TanayParikh can you confirm that's true?

Reply:
Here are the original PRs where I:

Moved to 3.14: https://github.com/dotnet/aspnetcore/pull/36845
Updated to a 3.14 queue w/ Python: https://github.com/dotnet/aspnetcore/pull/36929
- Associated Helix-Matrix: https://dev.azure.com/dnceng/public/_build/results?buildId=1384007&view=results
Reverted back to 3.12 after persistent failures: https://github.com/dotnet/aspnetcore/pull/36951

The linked helix-matrix might have info that isn't in the PRs - all I remember is that the failure I mentioned in the issue was persistent. I couldn't find anything more in emails.

Reply:
Maybe @HaoK remembers something I don't

Reply:
>  believe the only place that's running now is in the components pipeline.

The Components pipeline mainly runs Microsoft.AspNetCore.Components.E2ETests and those rely on Selenium, not Playwright. In addition, those tests don't run on Helix.

On the other hand, Playwright is used only in BlazorTemplates.Tests. Those tests are all quarantined and never run on any Linux e.g. from a recent `aspnetcore-quarantined-tests` run of 'main':

![image](https://user-images.githubusercontent.com/6431421/142129515-2c88a8d8-aa50-4f13-a9f3-effcc05d9b10.png)

In general, can that restriction be removed or reduced (adding a Linux platform)❔

Reply:
> In general, can that restriction be removed or reduced (adding a Linux platform)❔

@javiercn and @dotnet/aspnet-blazor-eng (because you're likely the most familiar w/ the `$(TestDependsOnPlaywright)` exclusions) please chime in here. I see Playwright is fully supported on Ubuntu 18.04 and 20.04 but we're running our Playwright tests on neither. I haven't searched enough to understand why Ubuntu 18.04 is also not running Playwright tests.

Reply:
@javiercn @dotnet/aspnet-blazor-eng any thoughts on Doug's question above?

Reply:
I'm now leaning toward closing everything but the duplicate Fedora testing unchanged and getting this in tonight or tomorrow. The experts can deal with our limited coverage for the Blazor template tests and Playwright system compatibility.